api/src/main/java/openconsensus/metrics/data/Value.java

```diff
@@ -86,6 +86,54 @@ public static Value summaryValue(Summary value) {
     return ValueSummary.create(value);
   }
 
+  /**
+   * Returns the double value of this {@link Value}.
+   *
+   * <p>This method should only be called with {@link Type#DOUBLE} {@link Value}.
+   *
+   * @return the double {@code Value}.
+   * @since 0.1.0
+   */
+  public double getDouble() {
```

Comment:
To be consistent with AttributeValue I would suggest getDoubleValue()

Reply:
SG, updated.